src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -0,0 +1,410 @@
+            # Phase 4: Induction - Verify answer
+            logger.info("Running Induction phase")
+            induction_result = await self._run_induction_phase(
+                full_input, core_question, result.final_answer, max_retries
+            )
+            result.verification_examples = induction_result["examples"]
+            result.verification_conclusion = induction_result["conclusion"]
+            result.total_llm_cost += induction_result["cost"]
+            result.phase_results["induction"] = induction_result
+            
+        except Exception as e:
+            logger.error(f"QADI cycle failed: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)